web/massastation/src/pages/Store/Install.tsx

```diff
@@ -80,25 +83,21 @@ function Install({
               <div className="mas-caption text-neutral mb-3">
                 {Intl.t('store.sidepanel.subtitle')}
               </div>
-              {isLoading ? (
-                <div className="mas-body2 mb-4 text-neutral">
-                  {Intl.t('store.loading')}
-                </div>
-              ) : (
-                <>
-                  <Input
-                    placeholder={Intl.t('store.sidepanel.placeholder')}
-                    name="url"
-                    value={url}
-                    onChange={(e) => handleChange(e)}
-                    error={error}
-                    customClass="bg-primary mb-3"
-                  />
-                  <Button type="submit" customClass="mt-3">
-                    {Intl.t('store.sidepanel.button')}
-                  </Button>
-                </>
-              )}
+              <>
+                <Input
+                  placeholder={Intl.t('store.sidepanel.placeholder')}
+                  name="url"
+                  value={url}
+                  onChange={handleChange}
+                  error={error}
+                  customClass="bg-primary mb-3"
+                />
+
+                <Button type="submit" customClass="mt-3" disabled={isLoading}>
+                  {isLoading && <Spinner variant="button" />}
+                  {Intl.t('store.sidepanel.button')}
+                </Button>
```

Comment:
Great!

That means possible we need to remove the pre/pos icon concept then...

Reply:
... and check if all goes fine where they are being used, of course. But all that we adresse in a specific MR!

Reply:
Feel free to close and merge!

Reply:
i'll check this after the layout refactor :)